src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,162 @@
+
+## Span
+
+The **span** is the primary building block of a distributed trace, representing an individual unit of work done in a distributed system. It can be the processing of an HTTP request by a server, the querying of a database, or an HTTP call to an external service.
+
+An example of a span that describes a call to an external service:
+
+```json
+    {
+      "trace_id": "a0fa8803753e40fd8124b21eeb2986b5",
+      "parent_span_id": "9c2a6db8c79068a2",
+      "span_id": "8c931f4740435fb8",
+      "start_timestamp": 1563890702.134,
+      "same_process_as_parent": true,
+      "description": "http://httpbin.org/base64/aGVsbG8gd29ybGQK GET",
+      "tags": { "http.status_code": 200, "status": "ok" },
+      "timestamp": "1563890702.934",
+      "op": "http",
+      "data": {
+        "url": "http://httpbin.org/base64/aGVsbG8gd29ybGQK",
+        "status_code": 200,
+        "reason": "OK",
+        "method": "GET"
+      }
+    }
+```
+
+When you initialize and load a Sentry SDK that supports tracing, it will automatically instrument parts of your running application, such as function calls and network operations. Instrumenting a database call could entail these three steps or subroutines:
+
+1. Calling a function to compose a database query
+2. Sending a query to a database
+3. Receiving and transforming results
+
+Each of these subroutines takes some time to perform (no matter how fast they could run). Sentry records these subroutines as a unit of work in the form of **spans**. Sentry can also attach useful information to spans, such as tags, additional contextual data, and a status to indicate if the subroutine failed or not.
+
+Spans can have descendant spans (or child spans). In our earlier example, the three subroutines can be spans within a larger encompassing span (the database call).
+
+Any instrumented span is part of a trace (identified by its trace id, `trace_id`), and each span has its own marker called the `span_id`, a fixed length of alphanumeric characters.
+
+### Hierarchy
+
+Each span can be connected to other spans using a parent-child hierarchy. 
+
+For example, a span called `updateJiraTask` would have child spans like `Check permissions`, `Update Task`, `sql.update`, `send.webhooks`.
+
+Each sub-task span would have an id in the attribute `parent_span_id` id of the `updateJiraTask` span.
+
+### Orphan Spans
+
+Orphan spans refer to spans that exist or are visible, but their parent spans are not visible or are not within Sentry's database. This could imply a bug in the Sentry application. If you're confident about this, please contact support: [support@sentry.io](mailto:support@sentry.io)
+
+Another reason could include that instrumented spans may not make it back to Sentry for processing. As an example, let us refer back to our earlier example of the web application consisting of:
```

Comment:
```suggestion
Another reason could include that instrumented spans may not make it back to Sentry for processing. As an example, let us consider the web application consisting of:
```

@MimiDumpling updating this since this is now in the glossary.